content/_data/changelogs/weekly.yml

```diff
@@ -12284,6 +12284,62 @@
     # pull: 5661 (PR title: Add missing `@since`)
     # pull: 5667 (PR title: Increase timeout for branch)
 
+  - version: '2.308'
+    date: 2021-08-24
+    changes:
```

Comment:
can you re-run generation please? there's some new PRs merged

Reply:
Sure thing!